jamaljsr/polar
# Add ability to view live logs from nodes

### Description

This PR adds the ability to view the live node logs in the UI.

### Steps to Test

1. Start your network
2. Right-click on one of the nodes
3. Click "View Logs" in the menu
4. Confirm that the logs are opened in a new window
5. Perform an action on the network, like mine a block
6. Confirm that new log statements are displayed in the window

### Screenshots

![image](https://user-images.githubusercontent.com/1356600/76035775-48941080-5f10-11ea-868c-b4c2eb054303.png)

![image](https://user-images.githubusercontent.com/1356600/76035851-85600780-5f10-11ea-9ea7-240aac01367c.png)
